client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -60,35 +87,85 @@ class DailyBackupStatus extends Component {
 		);
 	}
 
-	renderFailedBackup() {
-		const { backupAttempts, translate } = this.props;
+	renderFailedBackup( backup ) {
+		const { translate, timezone, gmtOffset } = this.props;
 
-		const hasBackupError = backupAttempts.error.length;
-		const errorMessage =
-			hasBackupError && backupAttempts.error[ 0 ].activityDescription[ 0 ].children[ 0 ].text;
+		const backupTitleDate = this.getDisplayDate( backup.activityTs );
+		const backupDate = applySiteOffset( backup.activityTs, { timezone, gmtOffset } );
+
+		const displayDate = backupDate.format( 'L' );
+		const displayTime = backupDate.format( 'LT' );
 
 		return (
 			<Fragment>
 				<Gridicon icon="cross-circle" className="daily-backup-status__gridicon-error-state" />
-
-				<div className="daily-backup-status__date">{ translate( 'Backup error' ) }</div>
+				<div className="daily-backup-status__date">{ translate( 'Backup attempt failed' ) }</div>
+				<div className="daily-backup-status__date">{ backupTitleDate }</div>
 				<div className="daily-backup-status__label">
-					{ hasBackupError ? errorMessage : translate( 'This day has no backups.' ) }
+					<p>
+						{ translate(
+							'A backup for your site was attempted on %(displayDate)s at %(displayTime)s and was not able to be completed.',
+							{ args: { displayDate, displayTime } }
+						) }
+					</p>
+					<p>
+						{ /* todo: Add the link of the guide: "backups help guide" */ }
+						{ translate(
+							'Check out the backups help guide or contact our support team to resolve the issue. View to get the issue resolved'
+						) }
+					</p>
+					<Button className="daily-backup-status__download-button">
+						{ translate( 'Contact support' ) }
+					</Button>
 				</div>
 			</Fragment>
 		);
 	}
 
-	render() {
-		const { backupAttempts } = this.props;
-		const dateHasGoodBackup = backupAttempts.complete.length;
+	renderNoBackups() {
+		const { translate } = this.props;
 
 		return (
-			<div className="daily-backup-status">
-				{ dateHasGoodBackup ? this.renderGoodBackup() : this.renderFailedBackup() }
-			</div>
+			<Fragment>
+				<Gridicon icon="sync" />
+
+				<div className="daily-backup-status__date">Next backup at...</div>
+				{ /* translate( 'This day has no backups.' ) */ }
+				<Button className="daily-backup-status__download-button" disabled={ true }>
+					{ translate( 'Download backup' ) }
+				</Button>
+				<Button className="daily-backup-status__restore-button" disabled={ true }>
+					{ translate( 'Restore to this point' ) }
+				</Button>
+
+				<div>
+					<strong>{ translate( 'Get Real-Time Backups' ) }</strong>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Get real-time backups' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-cloud-backup-status-with-selected-backup-plus-timezone)